autosklearn/ensembles/ensemble_selection.py

```diff
@@ -194,7 +194,8 @@ def _bagging(self, predictions, labels, fraction=0.5, n_bags=20):
         return np.array(order_of_each_bag)
 
     def predict(self, predictions):
-        for i, weight in enumerate(self.weights_):
+        non_null_weights = (weight for  weight in self.weights_ if weight > 0)
```

Comment:
I tried this solution at the beginning, but i had an "out of index" issue since predictions array contains only selected pipelines predictions.  
I tested this solution for different ensembles and it seems working. Models with a weight of zero are skipped because i'm looping just on `non_null_weights`.

Reply:
Oh, is that because of replacing `get_model_identifiers` with `get_model_selected_identifiers`?